src/main/java/ca/bc/gov/ag/efax/pdf/config/PdfConfiguration.java

```diff
@@ -3,20 +3,18 @@
 import org.springframework.boot.context.properties.EnableConfigurationProperties;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
-
-import ca.bc.gov.ag.outputservice.OutputServiceUtils;
+import org.springframework.oxm.jaxb.Jaxb2Marshaller;
 
 @Configuration
 @EnableConfigurationProperties(PdfProperties.class)
 public class PdfConfiguration {
 
-	@Bean
-	public OutputServiceUtils outputServiceUtils(PdfProperties pdfProperties) {
-		OutputServiceUtils outputServiceUtils = new OutputServiceUtils(
-				pdfProperties.getEndpoint(), 
-				pdfProperties.getUsername(), 
-				pdfProperties.getPassword());
-		return outputServiceUtils;
-	}
-	
+    @Bean
+    public Jaxb2Marshaller pdfMarshaller() {
+        Jaxb2Marshaller marshaller = new Jaxb2Marshaller();
+        // this package must match the package in the <generatePackage> specified in pom.xml
```

Comment:
what happens if it doesn't match?

Reply:
The package locations need to match (of where the model is stored and how it used) or the marshaller won't know where the model is when serializing/deserializing objects from the remote webservice.